common/app/experiments/Experiments.scala

```diff
@@ -79,7 +79,7 @@ object PoorDeviceConnectivity
       name = "poor-device-connectivity",
       description = "Reduce the load of the site for users experiencing poor device connectivity",
       owners = Seq(Owner.withEmail("open.journalism@theguardian.com")),
-      sellByDate = LocalDate.of(2023, 4, 4),
+      sellByDate = LocalDate.of(2023, 4, 12),
```

Comment:
Removed entirely in #26040

Reply:
I thought I'd give you some time to remove it but happy to revert this change